cypress/integration/proposal_esi.ts

```diff
@@ -1,8 +1,16 @@
 import faker from 'faker';
 
-const coProposerName = 'Benjamin';
-const coProposerEmail = 'ben@inbox.com';
-const visitorEmail = 'david@teleworm.us';
+const coProposer = {
+  id: 4,
+  name: 'Benjamin',
+  email: 'ben@inbox.com',
+  password: 'Test1234!',
+};
+const visitor = { id: 6, email: 'david@teleworm.us', password: 'Test1234!' };
+const PI = { id: 1, email: 'Javon4@hotmail.com', password: 'Test1234!' };
+const existingProposalId = 1;
+const acceptedStatusId = 1;
+const existingScheduledEventId = 996;
```

Comment:
I wonder if information that is in the DB already, could be shared somehow between tests